argilla/src/argilla/settings/_templates/_features.py

```diff
@@ -0,0 +1,166 @@
+# Copyright 2024-present, Argilla, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from enum import Enum
+from typing import Optional
+
+from datasets import ClassLabel, Features, Value, Sequence, Dataset
+
+from argilla.settings._templates._settings import DefaultSettingsMixin
+
+
+class FeatureType(Enum):
+    document_classification = "document_classification"
+    response_ranking = "response_ranking"
+    response_rating = "response_rating"
+
+
+class FeaturePresetsMixin(DefaultSettingsMixin):
+    @staticmethod
+    def document_classification(labels: list[str]):
+        labels.sort()
+        return [
+            Features(
+                {
+                    "text": Value("string", id=None),
+                    "label": ClassLabel(names=labels, id=None),
+                }
+            ),
```

Comment:
```suggestion
            Features(
                {
                    "text": Value("string", id=None),
                    "label": ClassLabel(names=labels, id=None),
                }
            ),
            Features(
                {
                    "text": Value("string", id=None),
                    "labels": Sequence(ClassLabel(names=labels, id=None)),
                }
            ),
```
Should we add something like this for multi-label?